Terminal.Gui/ConsoleDrivers/NetDriver/NetMainLoop.cs

```diff
@@ -47,55 +46,37 @@ void IMainLoopDriver.Setup (MainLoop mainLoop)
         Task.Run (NetInputHandler, _inputHandlerTokenSource.Token);
     }
 
-    void IMainLoopDriver.Wakeup () { _eventReady.Set (); }
+    void IMainLoopDriver.Wakeup () { }
```

Comment:
An event is ready when the queue has elements. 

If this is wrong how come the driver works?

Reply:
The queue has no elements of timeout and idle handler's. How it can return from the Take? Only by call the _eventsReady.Set the method returns.

Reply:
You haven't problems with your implementation because you are always send keystroke but if there's no input at all and you run for e.g. the Threadings scenario you will understand what I'm meaning.